### ISSUE TITLE
如何获取模型的中间变量?

### ISSUE THREAD

**Issue by zhoulinjun1994** [huawei-clouds/modelarts-example#57]
在API文档中描述 logits, end_points = mox.get_model_fn(...)(images) 时说返回值logits, end_points均为特征图，但实际用的时候发现logits是一个tensor，而end_points是一个字符串list，那如果我想获取预置模型中间层feature，应该怎么操作？

**Comment by zhoulinjun1994** [comment]
发现了一个好方法，用tensorflow里面的tf.get_default_graph().get_operations()函数，不过还是希望能用原生的moxing api，毕竟这种方法太麻烦了...
